cli/cmd/agent.go

```diff
@@ -213,6 +295,53 @@ func init() {
 	agentInstallCmd.Flags().BoolVar(&agentCmdState.InstallTrustHostKey,
 		"trust_host_key", false, "automatically add host keys to the ~/.ssh/known_hosts file",
 	)
+
+	// 'agent ctf aws ec2ic' flags
+	agentCTFAWSEC2ICCmd.Flags().StringVar(&agentCmdState.CTFInfraTagKey,
+		"infra_tag_key", "", "only install agents on infra with this tag key set",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringArrayVar(&agentCmdState.CTFInfraTag,
+		"infra_tag", []string{}, "only install agents on infra with this tag",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringVar(&agentCmdState.InstallAgentToken,
+		"token", "", "agent access token",
+	)
+	agentCTFAWSEC2ICCmd.Flags().BoolVar(&agentCmdState.InstallTrustHostKey,
+		"trust_host_key", false, "automatically add host keys to the ~/.ssh/known_hosts file",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringArrayVarP(&agentCmdState.CTFIncludeRegions,
+		"include_regions", "r", []string{}, "list of regions to filter on",
+	)
+
+	// 'agent ctf aws ssh' flags
+	agentCTFAWSSSHCmd.Flags().StringVar(&agentCmdState.CTFInfraTagKey,
+		"infra_tag_key", "", "only install agents on infra with this tag key set",
+	)
+	agentCTFAWSSSHCmd.Flags().StringArrayVar(&agentCmdState.CTFInfraTag,
+		"infra_tag", []string{}, "only install agents on infra with this tag",
```

Comment:
I was going to suggest `filter instances using this tag`, but perhaps instead we could say something like this?

```suggestion
		"infra_tag", []string{}, "only select instances with this tag",
```